raiden/accounts.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, keystore_path=None):
                 files = os.listdir(self.keystore_path)
             except OSError as ex:
                 msg = 'Unable to list the specified directory'
-                log.error('%s %s: %s', msg, self.keystore_path, ex)
+                log.error('%s %s: %s' % (msg, self.keystore_path, ex))
```

Comment:
I think this kind of replacement isn't a good idea, if any formatting error happens at runtime, this will throw before going inside the `try`ed `log` call. Instead of replacing it with it's own formatting, I'd suggest replacing it with keyword-based parameters (`log.error('OSError', msg=msg, path=self.keystore_path, ex=ex)`), or easier (for now) just adding some logging-compatibility setup: look [here](http://www.structlog.org/en/stable/standard-library.html), specially [in this paragraph](http://www.structlog.org/en/stable/standard-library.html#rendering-using-logging-based-formatters), and keeping (for now) all current [s]logging `log` calls compatible.

Reply:
PS: the quited paragraph's suggested setup may require still calling `logging.basicConfig` to enable its backend.